lib/deploy/events/apiGateway/cors.js

```diff
@@ -24,6 +25,16 @@ module.exports = {
         'Access-Control-Allow-Credentials': `'${config.allowCredentials}'`,
       };
 
+      // Enable CORS Max Age usage if set
+      if (_.has(config, 'maxAge')) {
```

Comment:
maxAge feature is great!
However, does this need for authorizer?
If not, would be great if you can separate maxAge PR from this one.